### PR TITLE
Turn GPS Base Stations

### PR DESCRIPTION
**Pull request by miriamseely**
Current metadata draft for the [Utah TURN GPS Stations ](https://opendata.gis.utah.gov/datasets/utah::utah-turn-gps-stations/about )layer. Let me know what you think! Thank you!

### CODE REVIEW

**Comment by DenisePeterson** [comment]
I think this looks good for the most part. One minor change:

"In some areas **we extends** past the boundary of Utah when we have been invited by those communities."

I would consider revising this. Perhaps:

"In some areas, our boundary extends beyond Utah where we have been invited..."

**Comment by miriamseely** [comment]
> I would consider revising this. Perhaps:
> 
> "In some areas, our boundary extends beyond Utah where we have been invited..."

That sentence isn't in the most recent commit.  At the moment, the language is simply: "This network is available in Utah and some surrounding areas". Do you want me to include the bit about us being invited, or just leave it at that? Either way works for me, I just wasn't sure if the being invited part was an essential detail or not.

**Comment by DenisePeterson** [comment]
@miriamseely - leave it the way you have it. I think it sounds good.

File: metadata/Cadastre/turn_gps_stations.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+This dataset serves as a general reference for both TURN GPS users and the general public. For more information on TURN GPS or to learn more on how to use the network, please visit [our website](https://turngps-billpay.ugrc.utah.gov/).
+
+### What does the dataset represent?
+
+Each point in this dataset represents the approximate location of a GPS base station. Features in this dataset include the station name, status, and coordinates.
+
+### How was the dataset created?
+
+<!--- I'm assuming we created this one? Did we just add new points to the dataset as stations were established, or was there a pre-existing of stations we added, or? --->
```
Comment (DenisePeterson) [inline metadata/Cadastre/turn_gps_stations.md:33]: That is correct. Sean provides new points as stations are added.

**Review by DenisePeterson — APPROVED** [review]

**Pull request merged**